src/components/Editors/Table/Table.tsx

```diff
@@ -93,6 +94,9 @@ export default function Table(props: TableProps) {
     const columns = []
     for (const [key, field] of Object.entries(fields)) {
       const columnIndex = parseInt(key) + 1
+      // Otherwise the _rowNumber and _rowValid are displayed on the table
```

Comment:
@roll without that the colum _rowNumber and _rowValid were being displayed and the DataGrid complained because there was two columns _rowNumber

Reply:
Currently, it's not really important I'm going to fix the Table